run_dir/design/flowcell.html

```diff
@@ -67,8 +79,27 @@
                         title="Treshold: {{ treshold }} million">{% for thousand in lane.get('clustersnb', '').split(',') %}<span class="thousand_group">{{ thousand }}</span>{% end %}
                     </td>
                     <th>%&nbsp;bases&nbsp;>&nbsp;Q30:</th>
-                    <td class="text-left {% if lane.get('overthirty', 0) < 30 %} danger {% elif lane.get('overthirty', 0) < 80 %}
-                      warning {% else %} success {% end %}">{{ lane.get('overthirty', 0) }}
+                    {% if flowcell.get('run_mode', '') == 'HiSeq X' %}
+                      {% set q30_treshold = 85.0 %}
+                    {% else %}
+                      {% set run_setup = flowcell.get('run_setup', '').split('x')[-1] %}
+                      {% try %} {% set run_setup = int(run_setup) %} {% except %}{% end %}
+                      {% if run_setup >= 250 %}
+                        {% set q30_treshold = 60.0 %}
+                      {% elif run_setup >= 150 %}
+                        {% set q30_treshold = 70.0 %}
+                      {% elif run_setup >= 100 %}
+                        {% set q30_treshold = 75.0 %}
+                      {% elif run_setup < 100 %}
+                        {% set q30_treshold = 80.0 %}
+                      {% end %}
+                    {% end %}
+                    {% set lane_q30 = lane.get('overthirty', 0.0) %}
+                    {% try %}{% set lane_q30 = float(lane_q30) %}{% except %}{% end %}
+                    <td class="text-left {% if lane_q30 < 40.0 %} danger
+                          {% elif lane_q30 < q30_treshold %} warning
+                          {% else %} success {% end %}" data-toggle="tooltip" data-placement="bottom"
+                        title="Treshold: {{ q30_treshold }}">{{ lane_q30 }}
```

Comment:
It's a bit of a weird word. _Thresh_ is an old-english word for straw / hay (_threshing_ is the action of removing the grain from wheat). People used to put it on their dirt floors inside, the _thresh hold_ was a piece of wood in the doorway that stopped it being kicked outside. That got shortened and _threshold_ became a term to describe the boundary of the house / boundaries in genereal.

There's your random trivia for the day ;)